README.md

```diff
@@ -108,8 +108,37 @@ dev.ror.192.168.1.5.xip.io or ror.192.168.1.5.xip.io
 
 ## Remote Debugging
 
-//TODO(duc4nh): https://github.com/acme101/ror-starter/issues/2
+We use RubyMine (IntelliJ) to run the Rails remote debugging in a Docker container.
+Firstly, starting remote debug session on the remote host:
 
+```
+$ docker-compose run --rm -p 1234:1234 -p 3000:3000 app-dev bundle exec rdebug-ide --host 0.0.0.0 --port 1234 --dispatcher-port 26162 -- /opt/app/bin/rails s -b 0.0.0.0 -p 3000 -e development
```

Comment:
dont' run a new container like, "docker-compose up -d app-dev" should have remote debugging support